shared/output/cloud/language_v1beta1/lib/google/cloud/language/v1beta1/language_service/client.rb

```diff
@@ -546,13 +561,19 @@ def annotate_text request, options = nil
             # @!attribute [rw] quota_project
             #   A separate project against which to charge quota.
             #   @return [::String]
+            # @!attribute [rw] universe_domain
+            #   The universe domain within which to make requests. This determines the
+            #   default endpoint URL. The default value of nil uses the environment
+            #   universe (usually GDU).
+            #   @return [::String,nil]
             #
             class Configuration
               extend ::Gapic::Config
 
+              # @private Deprecated and unused
```

Comment:
Still used in ` (@config.endpoint == Configuration::DEFAULT_ENDPOINT &&` so maybe phrase this differently?

Reply:
done